userland/tools/uncrustify/uncrustify.cfg

```diff
@@ -1,5 +1,111 @@
 # Uncrustify-0.65_f
 
+# ====================================================================
```

Comment:
Problems with history again. If you check the list of commits for this PR, you'll see that it includes one of Pat's, which is why it thinks this file is modified. Also, this PR can't even be merged right now because it conflicts with master, probably the same reason. I'm not sure what commands you're running to merge master, but something is wrong with them.

You should pull your master up to date and `git rebase` on top of it.